tests/e2e/tier-0.fmf

```diff
@@ -0,0 +1,35 @@
+summary: Tier 0 - QM Interconnect
```

Comment:
On the off-chance a non-expert looks at this: I would suggest adding a comment at the start of this file with a link to the documentation for the file format.

Reply:
Ack

Reply:
Ack, see README. md UPDATE in the PR